mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule RedactEx.MixProject do
   def project do
     [
       app: :redact_ex,
-      version: get_version(System.get_env("DRONE_TAG")),
+      version: "0.1.3",
```

Comment:
could this benefit from a [`.versions.yml`](https://github.com/primait/suite_py/blob/master/docs/index.md#i-want-to-bump-the-project-version) for `suite-py bump`?

Reply:
didn't know about it! However I don't see it used in other prima public libraries, am I wrong?

Reply:
(looking at amqpx, prima_ex_logger)

Reply:
I guess they just haven't adopted it yet - I just [added it to Veil](https://github.com/primait/veil/pull/41/commits/4bd65434c306e7683a9d71167e0f9ec97f33ae8b)

Reply:
nice, added! thanks!